src/seedu/addressbook/AddressBook.java

```diff
@@ -31,6 +32,10 @@
  **/
 public class AddressBook {
 
+    private static final String FILE_EXTENSION_DELIMETER = ".";
```

Comment:
I don't think this needs to be declared here. It's used in only one place and it's meaning is clear.